charts/sonarqube/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
-version: 1.6.5
-appVersion: 9.2.4
+version: 1.6.6
```

Comment:
i don't think an application version bump is a patch release :sweat_smile: 
as there are a few changes in this release that are user facing, i think this even justifies a major version bump. wdyt?

Reply:
Works for me 👍